### PR TITLE
Readme | update yml | included CONTRIBUTING

### PR DESCRIPTION
**Pull request by mergenthaler**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* Do not mix style changes/fixes with "functional" changes. It's very difficult to review such PRs and it most likely get rejected.
+* Do not add/remove vertical whitespace. Preserve the original style of the file you edit as much as you can.
+* Do not turn an already submitted PR into your development playground. If after you submitted PR, you discovered that more work is needed - close the PR, do the required work and then submit a new PR. Otherwise each of your commits requires attention from maintainers of the project.
+* If, however, you submitted a PR and received a request for changes, you should proceed with commits inside that PR, so that the maintainer can see the incremental fixes and won't need to review the whole PR again. In the exception case where you realize it'll take many many commits to complete the requests, then it's probably best to close the PR, do the work and then submit it again. Use common sense where you'd choose one way over another.
+
+### Local setup for working on a PR
+
+#### 1. Clone the repository
+* HTTPS: `git clone https://github.com/Nixtla/neuralforecast.git`
```
Comment (jmoralez) [inline CONTRIBUTING.md:31]:
```suggestion
* HTTPS: `git clone https://github.com/Nixtla/statsforecast.git`
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* SSH: `git clone git@github.com:Nixtla/neuralforecast.git`
```
Comment (jmoralez) [inline CONTRIBUTING.md:32]:
```suggestion
* SSH: `git clone git@github.com:Nixtla/statsforecast.git`
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* GitHub CLI: `gh repo clone Nixtla/neuralforecast`
```
Comment (jmoralez) [inline CONTRIBUTING.md:33]:
```suggestion
* GitHub CLI: `gh repo clone Nixtla/statsforecast`
```

**Comment by FedericoGarza** [comment]
@all-contributors pleas add @mergenthaler for code

**Comment by jmoralez** [comment]
This contributing points to the wrong repo btw

**Comment by FedericoGarza** [comment]
Right! I'll fix it in a new PR :)

**Pull request merged**